# yaw should be in radians, and tofs should be in meters
std_msgs/Header header
float64 yaw
float64[] tofs
